### PR TITLE
feat: use unified tabWidth for prettier autoformat

### PR DESCRIPTION
**Pull request by oclero**
We need a unified auto-formatting, especially for tabs/spaces in CSS. Otherwise we'll get undesired line changes in commits. Let's do it before the repository grows too much.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-module.exports =  {
-    singleQuote:  true,
-    jsxSingleQuote: true
+module.exports = {
+  singleQuote: true,
+  jsxSingleQuote: true,
+  tabWidth: 2,
```
Comment (maryiam) [inline .prettierrc.js:4]: I think that `tabWidth` and `useTabs` have these values (`2` and `false`) by default?
https://prettier.io/docs/en/options.html

Comment (oclero) [reply .prettierrc.js:4]: Ha yes you're right. Then... how can people commit files with 4 spaces?

Comment (maryiam) [reply .prettierrc.js:4]: Maybe it's because of the [`editorConfig`](https://editorconfig.org/) ?

Comment (oclero) [reply .prettierrc.js:4]: We could use that indeed. I can add it in this PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -103,11 +103,17 @@ All you need to do is :
...
 ### Good to know :
 
-- It's convenient to configure your ide in order automatically format your files. Please follow this [Editor Integration documentation](https://prettier.io/docs/en/editors.html).
+- It's convenient to configure your IDE in order to automatically format your files. Please follow this [Prettier Editor Integration documentation](https://prettier.io/docs/en/editors.html).
+
+- To ensure your IDE is already correctly configured, we use [EditorConfig](https://editorconfig.org/). Please install the required plugin for it, if your IDE needs one:
+
+  - [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)
+  - [Vim plugin](https://github.com/editorconfig/editorconfig-vim)
+  - WebStorm supports it natively.
```
Comment (maryiam) [inline CONTRIBUTING.md:116]: Apparently there is one https://plugins.jetbrains.com/plugin/7294-editorconfig 😛 (But it comes installed by default with webstorm)

**Pull request merged**